### ISSUE TITLE
Solar Fox - Inconsistent loading. Multiple restarts needed sometimes.

### ISSUE THREAD

**Issue by M-Walrus** [MiSTer-devel/Arcade-MCR1_MiSTer#11]
Game has graphics bugs when loading often times, that stops the game from starting.  Requires multiple restarts sometimes before it's playable.

**Comment by KremlingKuthroat19** [comment]
Yes. I've rebooted multiple times and the game has hideous graphical glitches. The background is completely green and has broken sprites.

What's interesting is that I paused the game for about 5 minutes and then unpaused and reset the game and everything looks perfect now. Then, I switched to another core and loaded this game, and everything works fine. Then, I did a hard reboot and everything works fine. Then I turned my MiSTer off and then turned it back on. It still works fine now.

This is good news, but maybe the game needs to build properly over a period of time? I saved my settings and am hoping that there won't be any more issues *fingers crossed*

I'll keep loading the game every once in a while and let you know if it stops working
